test/vue_phoenix/models/token_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule VuePhoenix.TokenTest do
+  use VuePhoenix.DataCase
+
+  alias VuePhoenix.{User, Token, Repo}
+
+  describe "changeset" do
+    @valid_attrs %{code: "code", user_id: 1}
+    @invalid_attrs %{code: ""}
+    @user_attrs %{email: "dungvt9691@gmail.com", password: "password", password_confirmation: "password"}
```

Comment:
Not ideal to use `module attribute`

can simply do simple assignment

`user_attrs = %{email: "dungvt9691@gmail.com", password: "password", password_confirmation: "password"}`

Reply:
If you want to create share test environemtn then can use `setup` block

For example:

```elixir
setup do
  user_attrs = %{email: "dungvt9691@gmail.com", password: "password", password_confirmation: "password"}
  [user_attrs: user_attrs]
end
```

Reply:
Then can be easily reuse in all blocks